vms/platformvm/block/executor/manager.go

```diff
@@ -124,6 +125,15 @@ func (m *manager) VerifyTx(tx *txs.Tx) error {
 		return ErrChainNotSynced
 	}
 
+	// If partial sync is enabled, this node isn't guaranteed to have the full
```

Comment:
just to double check - this function is only called at the transaction admission path, right? It's not part of the block processing logic, otherwise I guess we need some kind of pre-etna, post etna check?

Reply:
Correct. `VerifyTx` is not used during block execution. From the method comment:

```golang
// VerifyTx verifies that the transaction can be issued based on the currently
// preferred state. This should *not* be used to verify transactions in a block.
```

Reply:
oh, the interface method was not part of the diff so I didn't notice that.

Thanks.